### ISSUE TITLE
[FEAT]: Question about analyzing pictures.

### ISSUE THREAD

**Issue by zu0feng** [Mintplex-Labs/anything-llm#2550]
###Question

How can I use AnythingLLM analyze pictures?
Does it support this feature?

**Comment by MrSimonC** [comment]
Hey @zu0feng, I'd be happy to help you out. AnythingLLM can now analyze pictures, but it needs a bit of setup. The good news is that Ollama, a related project, has just released a pre-release version of 0.4.0 (https://github.com/ollama/ollama/releases/tag/v0.4.0-rc5). This new version allows you to download Llama 3.2 vision models, which enable image analysis.

To make it work, you'll need to run an Ollama instance on your local machine with this new version. Once you've got that set up, you can point AnythingLLM at your local Ollama instance. I've tried this out myself, and I was able to attach an image to a chat and ask AnythingLLM to describe it - and it worked!

So, to summarize: AnythingLLM can analyze pictures, but you'll need to set up an Ollama instance with the latest version and connect AnythingLLM to it. If you're new to this, it might take a bit of experimentation, but I hope this helps you get started!

**Comment by morbificagent** [comment]
Whats with external llms... Can AnythingLLM do this if they support pictures? I am using perplexity but it looks like no model is working with pics for me
